opal/browser/audio/param_schedule.rb

```diff
@@ -0,0 +1,24 @@
+module Browser; module Audio
+
+  class ParamSchedule
+    include Native
+
+    alias_native :value,               :setValueAtTime
```

Comment:
@meh mmm, in a way, haven't play much with these methods to be honest. Im wrapping a js library for creating tunes and they use it with different times but they have a common time factor: `when_time`, they use it like this:
```ruby
      scheduler.linear_ramp(current_note.frequency, when_time + start)
      scheduler.exponential_ramp(next_note.frequency, when_time + cutoff)
```